source/administration/bucket-replication/enable-server-side-multi-site-bucket-replication.rst

```diff
@@ -39,12 +39,18 @@ Multi-Site Active-Active replication configurations can span multiple racks, dat
 Requirements
 ------------
 
-Install and Configure ``mc`` with Access to Both Clusters.
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Access to All Clusters
+~~~~~~~~~~~~~~~~~~~~~~
 
-This procedure uses :mc:`mc` for performing operations on both the source and destination MinIO cluster. Install :mc:`mc` on a machine with network access to both source and destination deployments. See the ``mc`` :ref:`Installation Quickstart <mc-install>` for instructions on downloading and installing ``mc``.
+You must have access to all clusters to set up multi-site active-active bucket replication.
```

Comment:
```suggestion
You must have network access to all clusters to set up multi-site active-active bucket replication.
```

"access" is a little generic.

Really we mean both network access *and* IAM access (valid username/password) with all the permissions. Not sure how best to say that here.

Reply:
Also replace "Clusters" with "Deployments" throughout.

Though I suppose we should check internally - I am trying to avoid using "Clusters" since once you hit K8s things get weird with clusters in clusters